aqt/archives.py

```diff
@@ -257,7 +268,20 @@ def _parse_update_xml(self, archive_url, target_packages: Optional[ModuleToPacka
         # if we have located every requested package, then target_packages will be empty
         if len(target_packages) > 0:
             message = f"The packages {target_packages} were not found while parsing XML of package information!"
-            raise NoPackageFound(message)
+            raise NoPackageFound(message, suggested_action=self.help_msg(target_packages.get_modules()))
+
+    def help_msg(self, missing_modules: Iterable[str]) -> Iterable[str]:
+        base_cmd = f"aqt list-qt {self.os_name} {self.target}"
+        arch = f"Please use '{base_cmd} --arch {self.version}' to show architectures available."
+        mods = f"Please use '{base_cmd} --modules {self.version} <arch>' to show modules available."
```

Comment:
This particular error message won't make sense unless #401 is merged, because it asks the user to enter architecture as a parameter to `--modules`. I can change it if necessary, but I thought it would be easier to write it like this so we don't have to change it when #401 is merged.

**Note:** In the context of `QtArchives`, we could replace the `<arch>` term with `{self.arch}`, and the user could just copy and paste the resulting command. However, `SrcDocExamplesArchives` uses this code as well, and in `SrcDocExamplesArchives` the `self.arch` data member stores "src", "doc", or "example", which would not work as an arch in the `aqt list-qt` command. I wasn't sure about the best way to resolve this, so I left it as `<arch>`.